themes/default/content/blog/software-developer-experience-devex-devx-devops-culture/index.md

```diff
@@ -0,0 +1,263 @@
+
+## Is Developer Experience Important
+
+It is not just important. **It's business critical.** If your company is in the business of creating and selling software or relies on critical internal software, developers are vital internal stakeholders to your business foundation.
+
+Greg Mondello, director of product at GitHub, said, "In most contexts, software development capacity is the limiting factor for innovation. Therefore, improvements to the effectiveness of software development are inherently valuable [...] companies with better DevEx outperform their competitors, regardless of vertical."
+
+{{< figure alt="Platform Engineering Forrester Opportunity Snapshot - DevEx impacts overall business performance. Credit: Humanitec" src="/blog/software-developer-experience-devex-devx-devops-culture/what-impact-are-you-experiencing-resulting-of-improving-developer-experience.jpg" width=70%  caption="Platform Engineering Forrester Opportunity Snapshot - DevEx impacts overall business performance. Credit: Humanitec" >}}
+
+As per [McKinsey's report](https://www.mckinsey.com/capabilities/mckinsey-digital/our-insights/tech-forward/why-your-it-organization-should-prioritize-developer-experience), an enhanced Developer Experience (DevEx) can yield significant advantages for organizations, including enhanced talent hiring and retention, improved security, and increased developer productivity- this makes DevEx important for all companies, not just tech.
+
+A positive developer experience can yield numerous business advantages, which can include:
+
+1. **Improvement in Collaboration Across Departments**
+2. **Higher Attraction and Retention Top Talent**
+3. **Increase Developer Productivity and Developer Velocity**
+4. **Reduction in Operational Costs**
+5. **Accelerate Time-to-Market**
+6. **Support Innovation**
+7. **Elevate Customer Attraction and Retention**
+8. **Increase Bottom Line through Revenue Growth**
+
+To this end, let's look into what goes into developer experience and how to achieve it.
+
+## The GitHub's Developer Experience (DevEx) Formula
+
+GitHub's DevEx formula takes into consideration the following:
+
+- Productivity: The speed and simplicity of implementing changes to a codebase.
+- Impact: The seamless transition from idea to production without obstacles.
+- Satisfaction: The environment, workflows, and tools influence developer happiness.
+
+Optimizing DevEx involves fostering a collaborative environment where developers can be most productive, impactful, and satisfied.
+
+{{< figure alt="DevEx formula. Credit: GitHub" src="/blog/software-developer-experience-devex-devx-devops-culture/github-developer-experience-formula.png" width=100%  caption="DevEx formula. Credit: GitHub" >}}
+
+Noah Gift, the founder of Pragmatic AI Labs and a professor at Duke University specializing in machine learning, MLOps, AI, data science, and cloud architecture, noted that with the right platform, there could be:
+
```

Comment:
Get rid of the punctuation marks for the bullets here (and everywhere in the article)